pennylane/kernels/cost_functions.py

```diff
@@ -175,3 +176,59 @@ def circuit(x1, x2):
         rescale_class_labels=rescale_class_labels,
         normalize=True,
     )
+
+
+def task_model_alignment(
+    N,
+    task_weights,
+    kernel_evals,
```

Comment:
I now know that `evals` stands for "eigenvalues" but somehow I find it really confusing since in the ML world it's used a lot for "evaluations"

Reply:
good point, I was more inspired by numpy here...maybe `eigvals`?